config/pom.xml

```diff
@@ -112,10 +112,6 @@
                         <Export-Package>
                             org.axonframework.config.*
                         </Export-Package>
-                        <Import-Package>
-                            org.slf4j.*,
-                            *;resolution:=optional
```

Comment:
Removing the `*;resolution:=optional` line will make the bundle fail to start, as the Configuration API has a dependency on many components outside of the `org.axonframework.config` package. They are, however, all optional (as they depend on the actual configuration).
`org.slf4j` is indeed a currently unused import, although there is another ticket to add debug logging on the Configuration. That would bring the Import back....

Reply:
brought back import: *;resolution:=optional